android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.example.plugin_tozny">
-    <uses-sdk android:minSdkVersion="23"
+    <uses-sdk
```

Comment:
as a general rule we should have some minimum version, and specifically for the exact issue you ran into where register identity doesn't work on anything less than 26 (thus 26 seems like the natural minimum version)

Reply:
i will set the minimum to 26 in the build.gradle files. since that overrides the values set here in the manifest.